lib/broadside/deploy.rb

```diff
@@ -4,9 +4,11 @@ class Deploy
 
     attr_reader :target
 
+    delegate :tag, to: :target
+
     def initialize(options = {})
       @target = Broadside.config.get_target_by_name!(options[:target])
-      @tag = options[:tag] || @target.tag
+      @target.tag = options[:tag] if options[:tag]
```

Comment:
(that and i don't see why [this logic](https://github.com/lumoslabs/broadside/pull/91/files) needs to be changed at all?